source/release-notes/v1.7-release-notes.rst

```diff
@@ -79,9 +105,135 @@ Passenger has been upgraded from ``5.3.7`` to ``6.0.4``. NGINX has been upgraded
+scheme configured for the dashboard. 
+
+There's no additional configuration required for this other than what's documented 
+:ref:`here <branding>`.
+
+Maintenance mode
+................
+
+OOD can be put into 'maintenance mode' where it serves a static page to either all
+users or a subset (staff on a VPN for example).  This static page informs users
+that maintenance is underway while administrators perform disruptive tasks.
+
+See :ref:`these docs <maintenance-mode>` on how to configure and enable this feature.
+
+Titles for Files menu entries
+.............................
+
+The shortcuts in the files dropdown menu can now show an optional title along with the 
+directory path. The :ref:`original documentation <add-shortcuts-to-files-menu>` on this 
+feature has been updated to reflect the new options.
+
+
+Documentation for Keycloak as identity broker with CILogon and Keycloak with Duo 2FA
+....................................................................................
+
+Documentation for Keycloak as identity broker with CILogon has been added in the documentation
+entitled ':ref:`authentication-tutorial-oidc-keycloak-rhel7-configure-cilogon`'.
+
+Documentation for Keycloak with Duo two factor authentication has been updated in the documetnation
+entitled ':ref:`authentication-duo-2fa-with-keycloak`'.
+
+Add Grafana support for ActiveJobs app
+......................................
+
+OnDemand's ActiveJobs app can display graphs for jobs that are pulled from Grafana.
+Details on how to configure Grafana support are in :ref:`these docs <grafana-support>`.
+
+Improvements to ood-portal-generator
+....................................
+
+The ood-portal-generator app has been rewritten in Ruby and extensive testing added.
+Checksums generated for ``ood-portal.conf`` now exclude comments.
+
+Add an ssh wrapper to the shell app
+...................................
+
+An ssh wrapper script can now be used instead of just the ``ssh`` command in the
+shell app.  See how to configure this :ref:`here <ssh-wrapper>`.
+
+Regenerate ood-portal.conf with every apache restart
+....................................................
+
+Systemd will now regenerate the ood-portal.conf before every apache restart.  This means
+and administrators can now edit the ``ood_portal.yml`` configuration and then restart httpd
+with systemctl and apache will read the new configuration, because a new ood-portal.conf
+will have been written just before restarting.
+
+Support sanitizing job names
+............................
+
+Administrators can now set the ``OOD_JOB_NAME_ILLEGAL_CHARS`` environment variable to prevent
+characters from being used in job names.  For example if you do not want to use ``/`` in job
+names (as is the case with some :ref:`Grid Engine <resource-manager-sge>` versions) you would
+set ``OOD_JOB_NAME_ILLEGAL_CHARS: '/'`` in the ``pun_custom_env`` attribute of ``nginx_stage.yml``.
```

Comment:
Maybe we should have a section in "Customization" for "Sanitizing job names" and then reference that in the Grid Engine example? This is a good example of several cases where in the release notes (1.5, 1.6, and now this) we sometimes have better feature documentation than in the main documentation area.